src/dicomweb_client/session_utils.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import os
+from typing import Optional, Any
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+def create_session_from_auth(
+        auth: [requests.auth.AuthBase]) -> requests.Session:
+    '''
```

Comment:
Add a brief description of the subroutine performed by each utility function. Their functionality is pretty self-explanatory given the description of the parameters and return values, but it would be nice for completeness